src/main/java/com/o19s/es/ltr/feature/store/StoredLtrModel.java

```diff
@@ -85,11 +85,7 @@ public StoredLtrModel(StreamInput input) throws IOException {
         rankingModelType = input.readString();
         rankingModel = input.readString();
         modelAsString = input.readBoolean();
-        if (input.getTransportVersion().onOrAfter(TransportVersions.V_7_7_0)) {
```

Comment:
Elasticsearch 9.0.0 anyway only support more recent version of the transport, so the two branches do not make sense anymore.